flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java

```diff
@@ -62,21 +62,22 @@
   /**
    * Create an Iceberg {@link org.apache.iceberg.catalog.Catalog} loader to be used by this Flink catalog adapter.
    *
-   * @param name    Flink's catalog name
-   * @param options Flink's catalog options
+   * @param name       Flink's catalog name
+   * @param hadoopConf Hadoop configuration for catalog
+   * @param options    Flink's catalog options
    * @return an Iceberg catalog loader
    */
-  protected CatalogLoader createCatalogLoader(String name, Map<String, String> options) {
+  protected CatalogLoader createCatalogLoader(String name, Configuration hadoopConf, Map<String, String> options) {
```

Comment:
It's a little odd that the arguments here are the same as `createCatalog`, but that they are in a different order and use "options" rather than "properties". Should we at least use the same argument order?

Reply:
I will use same order and `options`.